### PR TITLE
Clarify that methods may have additional DID URI components

### PR DESCRIPTION
**Pull request by troyronda**

### CODE REVIEW

File: docs/spec/did-uri.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ Example of Short-Form DID URI:
 did:sidetree:EiDahaOGH-liLLdDtTxEAdc8i-cfCz-WUcQdRJheMVNn3A
 ```
 
+An implementor ****MAY**** define additional components in their method's DID URI composition.
+
```
Comment (OR13) [inline docs/spec/did-uri.md:20]: for example.... ?

Comment (troyronda) [reply docs/spec/did-uri.md:21]: @OR13 Added examples.

**Comment by troyronda** [comment]
@csuwildcat @OR13 added `cas` to `web` example: `webcas`.

**Review by OR13 — APPROVED** [review]

**Comment by troyronda** [comment]
Text added in #1053

**Pull request closed without merging**